manuscript/ha-docker-swarm/nodes.md

```diff
@@ -63,6 +63,19 @@ Set your local timezone, by running:
 ln -sf /usr/share/zoneinfo/<your timezone> /etc/localtime
 ```
 
+### Install Docker
+
+A fresh installation of Ubuntu does not include Docker, so you wil need to install it manually. Run the following command **on each node**
+
+```
+apt install docker.io -y
+```
+!!! tip
+    In order not to have to type `sudo` in front of every Docker command in this guide, you can add your user to the `docker` group by executing the following command **on each node**:
+    ```sudo usermod -aG docker ${USER}```
+    Once you do that, you'll need to log out and back in again for the new group permissions to become active.
+
+
```

Comment:
Added info for those new to this process that they'll need to manually install Docker. Also a tip to keep them from having to type `sudo` every time.